solvation_analysis/tests/test_basics.py

```diff
@@ -0,0 +1,87 @@
+import sys
+import MDAnalysis as mda
+import numpy as np
+import pytest
+
+from MDAnalysis.topology.tables import masses
+from solvation_analysis.tests.datafiles import (
+    bn_fec_data,
+    bn_fec_dcd_wrap,
+    bn_fec_dcd_unwrap,
+    bn_fec_atom_types,
+)
+
+
+def test_solvation_analysis_imported():
+    """Sample test, will always pass so long as import statement worked"""
+    assert "solvation_analysis" in sys.modules
+
+
+def make_grid_universe(n_grid, residue_size, n_frames=10):
+    """
+    Will make a universe of atoms with specified attributes.
+
+    Parameters
+    ----------
+    n_grid - dimension of grid sides
+    residue_size - size of mda residues
+    n_frames - number of frames
+
+    Returns
+    -------
+    A constructed MDanalysis.Universe
+    """
+    n_particles = n_grid ** 3
+    assert (
+        n_particles % residue_size == 0
+    ), "residue_size must be a factor of n_particles"
+    n_residues = n_particles // residue_size
+    atom_residues = np.array(range(0, n_particles)) // residue_size
+
+    grid = np.mgrid[0:n_grid, 0:n_grid, 0:n_grid]  # make the grid
+    frame = grid.reshape([3, n_particles]).T  # make it the right shape
+
+    traj = np.empty([n_frames, n_particles, 3])
+    for i in range(n_frames):
+        traj[i, :, :] = frame  # copy the coordinates to n frames
+    u = mda.Universe.empty(
+        n_particles, n_residues=n_residues, atom_resindex=atom_residues, trajectory=True
+    )  # jam it into a universe
+    u.load_new(traj)
+    u.add_TopologyAttr("masses", np.ones(n_particles))
+    u.add_TopologyAttr("resid")
+    return u
+
+
+@pytest.fixture
+def u_grid_3():
+    return make_grid_universe(6, 3)
+
+
+@pytest.fixture
+def u_grid_1():
```

Comment:
Probably worth a quick comment what this (and other) fixture is

Reply:
I added brief descriptors to the fixtures.